src/router/booking.ts

```diff
@@ -2,9 +2,9 @@ import { Router } from 'express';
 import bookingController from '../controllers/BookingController';
 
 export default (router: Router): void => {
-  router.get('/booking', bookingController.myBookings);
-
-  router.post('/booking', bookingController.booking);
-
-  router.get('/booking/price/:id', bookingController.price);
+  router.get('/booking/:address', bookingController.myBookings);
+  router.post(
```

Comment:
I guess we should rename this endpoint. For hotels, we have `guests` not `passengers`.
We should avoid endpoint syntax like `set-something` because the type of request `POST` already means - `set`, or `add` or `create`.